docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+### alternative tag is 'stable'
+ARG RUST_VSN='stable'
+
+##### Build
+FROM docker.io/clux/muslrust:${RUST_VSN} as builder
+
+COPY / ./
+RUN cargo build --release
+
+RUN mkdir -p /rootfs/etc/fpush \
+ && mv $(find target/ -name fpush -type f -executable) /rootfs/fpush \
+ && touch /rootfs/etc/fpush/settings.json
+
+##### Runtime
+FROM gcr.io/distroless/static-debian11:nonroot AS prod
```

Comment:
```suggestion
FROM gcr.io/distroless/static-debian12:nonroot AS prod
```

Reply:
ah, great. In July, `debian12` variants haven't been available yet!